### ISSUE TITLE
daprd does not install on container rebuild

### ISSUE THREAD

**Issue by sw34** [microsoft/vscode-dev-containers#888]
If you rebuild a dev container daprd fails to install into the container because the other dapr containers are already spun up (dapr_placement, dapr_redis, dapr_zipkin).  

- VSCode Version: 1.56.2
- Local OS Version: Windows 10/WSL Ubuntu 20.04
- Name of Dev Container Definition with Issue: dapr-dotnet

Steps to Reproduce:

1. create remote container, including building the image
2. edit any files in .devcontainer folder
3. rebuild dev container

Workaround:

run following commands in newly rebuilt container
```
dapr uninstall
dapr init
```

**Comment by Chuxel** [comment]
@philliphoff This is a good catch.

Should `postCreateCommand` perhaps be "dapr uninstiall && dapr init" to avoid that?

**Comment by philliphoff** [comment]
@Chuxel That would probably work, though if Dapr is otherwise setup it's just extra initialization time that isn't technically required.  What might be better still is a bit more nuanced logic in a real script that only attempts to do a `dapr init` when Dapr isn't already setup.

I had planned on moving these Dev Containers to a "Docker-in-Docker" model which simplifies other aspects of using Dapr, too; it would alleviate the problem altogether because it would *always* be starting from a known state (i.e. no Dapr), so the initialization logic would always succeed.  However, I got bogged down by attempts to make it work on the M1 Mac, which requires the base images be `arm64` compatible; I discovered that there was no Debian `arm64` build of Moby, needed as the base .NET images are based on Debian.  (Unlike the base Node.js images, which are based on Ubuntu, which can use Moby builds we (i.e. Microsoft) appear to build ourselves.  Or it might have been the inverse, I forget exactly.) In any case, my attention got drawn elsewhere and I've not made it back to updating the Dev Containers.
